schemata/export_files_check_batch.json

```diff
@@ -0,0 +1,103 @@
+{
+  "$schema": "http://json-schema.org/draft-04/hyper-schema",
+  "id": "/schemata/export_files_check_batch",
+
+  "title": "Export/Report Files (check status in batch)",
+  "stability": "production",
+  "description": "Closely mimics the \"Recent Files\" screen in the WebAdMIT application.  This call will display all export files that have recently run under a particular identity, and their status.  Importantly, the download URL is not displayed in this response as these individual expiring URLs can only be generated by calling for the status of an individual file directly using the previous call.  You can use the results of this call and loop through each one to get a download URL for each export file marked as \"Available\".",
+
+  "links": [
+    {
+      "description": "The response is the collection of id’s and statuses of the running exports for a specific identity within a user’s WebAdMIT profile.",
+      "href": "/api/v1/user_identities/:user_identity_id/export_files",
+      "method": "GET",
+      "rel": "instances",
+      "title": "List",
+      "http_header": { "x-api-key": "0123456789abcdef0123456789abcdef" },
+      "targetSchema": {
+        "properties": {
+          "href": {
+            "type": "string",
+            "description": "Hypertext reference to this resource.",
+            "pattern": "/api/v1/user_identities/\\d+/export_files",
+            "example": "/api/v1/user_identities/37442/export_files"
+          },
+          "export_files": {
+            "type": "array",
+            "items": {
+              "type": "object",
+              "required": ["id", "href", "export_id", "status"],
+              "properties": {
+                "id": {
+                  "type": "integer",
+                  "example": 74780,
+                  "description": "Unique identifier of this export file."
+                },
+                "href": {
+                  "type": "string",
+                  "description": "Hypertext reference to this resource.",
+                  "pattern": "/api/v1/exports/\\d+/export_files/\\d+",
+                  "example": "/api/v1/exports/86466/export_files/74780"
+                },
+                "export_id": {
+                  "type": "integer",
+                  "example": 86446,
+                  "description": "Unique identifier of the associated export."
+                },
+                "status": {
+                  "type": "string",
+                  "description": "Current status of this export file.",
+                  "enum": ["initializing", "queued", "in_progress", "available", "success_with_errors", "empty_list", "failed"],
```

Comment:
The PDF says this is `"Available"`, not `"available"`.  (I'm assuming that's wrong since you used this schema to test it.)